absl/time/time.h

```diff
@@ -313,12 +313,12 @@ class Duration {
 
 // Relational Operators
 
-#ifdef __cpp_impl_three_way_comparison
+#ifdef __cpp_lib_three_way_comparison
 
 ABSL_ATTRIBUTE_CONST_FUNCTION constexpr std::strong_ordering operator<=>(
```

Comment:
My reading is that
* `__cpp_impl_three_way_comparison` is required for `<=>`
* `__cpp_lib_three_way_comparison` is required for `std::strong_ordering`

So I think this technically needs both.

Reply:
Yes, technically.
But stl library is hardly capable of implementing the support if the compiler does not have it.
Hence I believe that checking `_cpp_lib` capability is enough (and the diff can be produced by a simple tool i. e. sed)